src/components/02-components/data-table/data-table.stories.tsx

```diff
@@ -44,3 +46,11 @@ export const Staff: StoryObj<typeof DataTable> = {
     filters: staffFilterModel,
   },
 };
+
+export const EnrolledStudents: StoryObj<typeof DataTable> = {
+  args: {
+    rows: enrolledStudentsTableRows,
+    columns: enrolledStudentsTableColumns,
+    toolbar: () => <GridToolbar title="Enrolled Students" />,
```

Comment:
We might need to build an alternative toolbar to support the "Create an Experience" button.  We can discuss this over zoom when you have time.

Reply:
Created a new toolbar that prints a button, and the text button and href button are arguments in order to allow reuse of the toolbar when it's necessary for other dataTables.

Reply:
I am going to create a tech debt ticket and we can address this in a later issue but currently this doesn't allow for selecting multiple students and passing them into the create an experience page as a GET parameter.